packages/plugin/src/theia.d.ts

```diff
@@ -1878,6 +1878,11 @@ declare module '@theia/plugin' {
          * not implemented yet
          */
         picked?: boolean;
+
+        /**
+         * Always show this item.
+         */
+        alwaysShow?: boolean;
```

Comment:
I don't see that it used anywhere in Theia? Why do we add it?

Reply:
@akosyakov https://github.com/Microsoft/vscode/blob/master/src/vs/vscode.d.ts#L1599-L1602

Reply:
and https://che-incubator.github.io/vscode-theia-comparator/status.html (alwaysShow was missing)

Reply:
@benoitf yes, but if we declare things without implementation what is the value of this status page then? I would prefer to see it is red till it is really supported.

Reply:
so `picked` should be implemented as well and not mocked

Reply:
> so picked should be implemented as well and not mocked

Exactly, i did not know that is not implemented, we could mock everything in one PR today and make the page green. The hardest part is to implement it. I like that the page shows our progress, also theia-vscodecov points to code where vscode extension won't work properly. 

Particularly in this case i also don't see why mocking is urgent, it is not even a function, i doubt it can lead to runtime errors.

Reply:
I'm fine with rejecting mocks now that we've a large codebase

Reply:
@akosyakov @benoitf sorry guys, I looked at `picked` when adding `alwaysShow` and didn't realize it was only a mock :( I'm sorry about that.

Reply:
@vince-fugnitto it's fine, what would be really cool if you figure out what `alwaysShow` should do and implement it for Theia!